Watchman.Discord/Areas/Users/Controllers/UsersController.cs

```diff
@@ -44,10 +49,24 @@ public void GetAvatar(DiscordRequest request, Contexts contexts)
         [DiscordCommand("add role")] //todo
         public void AddRole(DiscordRequest request, Contexts contexts)
         {
-            var commandRole = request.OriginalMessage.ToLowerInvariant().Replace("-add role ", string.Empty); //TODO use DiscordRequest properties
-            var safeRoles = this._queryBus.Execute(new GetDiscordServerSafeRolesQuery(contexts.Server.Id)).SafeRoles;
-            var messagesService = _messagesServiceFactory.Create(contexts);
-            _rolesService.AddRoleToUser(safeRoles, messagesService, contexts, commandRole);
+            var args = request.Arguments.Skip(1).ToList();
```

Comment:
to args nic nie mówi
nazwa zmiennej powinna mówić jasno co sie w niej znajduje 
w "args" to może być wszystko, tym bardziej to skip(1) jeszcze bardziej pogarsza czytelność

i czemu ToList()